docs/plasmapy_sphinx/__init__.py

```diff
@@ -67,7 +67,7 @@
 
 from sphinx.application import Sphinx
 
-from . import autodoc, automodsumm, directives, utils
+from docs.plasmapy_sphinx import autodoc, automodsumm, directives, utils
```

Comment:
- [ ] Revert everything from `plasmapy_sphinx`
 - [ ] Use the `exclude` thing to not have this hook do stuff in `docs/plasmapy_sphinx`